README.md

```diff
@@ -309,9 +309,21 @@ Ask your questions related to cactoos library on [Stackoverflow](https://stackov
 Just fork the repo and send us a pull request.
 
 Make sure your branch builds without any warnings/issues:
+```
+mvn clean verify -Pqulice
+```
 
+To run a build similar to the CI with Docker only, use:
 ```
-mvn clean install -Pqulice
+docker run \
+	--tty \
+	--interactive \
+	--workdir=/main \
+	--volume=$(shell pwd):/main \
+	--volume=cactoos-mvn-cache:/root/.m2 \
+	--rm \
+	maven:3-jdk-8 \
+	bash -c "mvn clean install site -Pqulice -Psite --errors"
 ```
```

Comment:
I think, we should also include `sudo chown -R $USER:$GROUP .`  because of  https://github.com/yegor256/cactoos/pull/1196#issuecomment-536206649